AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/consts/scripting_constants.py

```diff
@@ -0,0 +1,89 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Holds constants used across both hydra and non-hydra scripts.
+"""
+import os
+import azlmbr.paths as paths
+
+"""
+Constants for window and tab names
+
+"""
+SCRIPT_CANVAS_UI = "Script Canvas"
+NODE_PALETTE_UI = "Node Palette"
+NODE_INSPECTOR_UI = "Node Inspector"
+ASSET_EDITOR_UI = "Asset Editor"
+SCRIPT_EVENT_UI = "Script Events"
+VARIABLE_MANAGER_UI = "Variable Manager"
+
+"""
+Constants for Variable Manager
+
+"""
+RESTORE_DEFAULT_LAYOUT = "Restore Default Layout"
+
+
+"""
+String constants for Node Palette
+"""
+NODE_CATEGORY_MATH = "Math"
+NODE_STRING_TO_NUMBER = "String To Number"
+NODE_TEST_METHOD = "test_method_name"
+
+"""
+Constants for Node Inspector
+"""
+NODE_INSPECTOR_TITLE_KEY = "Title"
+"""
+Constants for Asset Editor
+"""
+SAVE_ASSET_AS = "SaveAssetAs"
+DEFAULT_SCRIPT_EVENT = "EventName"
+DEFAULT_METHOD_NAME = "MethodName"
+PARAMETER_NAME = "ParameterName"
```

Comment:
Create an `asset_editor` consts file and move these there.

Reply:
For any of these suggestions, let's approach this as a deprecation path.

Maybe created a "Deprecated" section with comments saying, "Use new version in x file."